department-of-veterans-affairs/caseflow
# Pamatya/appeals 33775

Resolves [APPEALS-33775](https://jira.devops.va.gov/browse/APPEALS-33775)

# Description
Please explain the changes you made here.
Add a condition to display No Recognized POA and No known POA for attorney Claimant. 
Added a condition to display No Recognized POA in Add issues. 
Added Spec test to 

## Acceptance Criteria
- [ ] On the ‘Add / Remove Issues’ screen, the following text displays in Claimant’s POA field:
Informational text: No recognized POA
- [ ] If an Unrecognized Claimant - 'Attorney' type is selected, the following text displays in Claimant's POA field:
- [ ]Unrecognized 'Name not listed' Attorney claimant
- [ ]Informational text: No recognized POA
- [ ]Selected from list/drop-down Attorney claimant
 -[ ] Informational text: No known POA

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
2.  Login as a ACBAUERVVHAH or any user that can do Intake and also select VHA benefit type. 
3.  Do HLR or SC intake
4. Enter all the fields in the field
   a) benefit type should be 'vha'
   b) Is the claimant Someone other than veteran? -> Yes
   c) Select -> Claimant not listed
   d) Click on "Continue to next step"
   e) Select -> Relationship to veteran - choose either child /Spouse
   f) fill first name and last name as those two are required.
   g) Do you have a VA Form 21-22 for this claimant? - select yes
   h) Click continue to next step -> notice Continue to next step is disabled.
   i) Representative's Name -> search for "name not listed" and select , notice -> continue to next step is enabled and banner 
    is displayed.
   k) click on continue to next step
   l) notice Review and confirm modal is displayed and click on confirm
   m) Once you click on the Confirm you will be redirected to Add Issue Page. 
    n) Confirm you are seeing No Recognized POA
5) Enter HLR Or Sc, repeat steps from 4(a -> d)
 e)  Select -> relationship to veteran -> choose -> attorney
f) under attorney -> once select -> Name not listed and continue to Add issues -> at Add/Remove issue page you should see 'No Recognized POA' 
g) next time select-> already present Attorney and continue to Add Issue -> this time you should see 'No Known POA'.
 



# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/122557351/68e43a45-961b-4496-88e8-408f25da2083)

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
